app/templates/contact.hbs

```diff
@@ -11,7 +11,7 @@
         class="au-c-link"
         target="_blank"
         rel="noopener noreferrer"
-      >handleiding</a>.
+      >handleiding (externe link)</a>.
```

Comment:
I've been thinking about creating a dedicated `ExternalLink` or `LinkExternal` component, do you think that is something that can be useful for other projects? In that case it might make sense to add it to ember-appuniversum instead.

Reply:
Yes, could be helpful as we use it in most projects.